pkg/kubelet/dockershim/docker_container.go

```diff
@@ -79,7 +79,8 @@ func (ds *dockerService) ListContainers(filter *runtimeApi.ContainerFilter) ([]*
 }
 
 // CreateContainer creates a new container in the given PodSandbox
-// Note: docker doesn't use LogPath yet.
+// Note: `LogPath` is not used because docker cannot store the log to an
```

Comment:
We are using `LogPath` right? We just make symlink, instead of creating actual log file.